ocp_utilities/monitoring.py

```diff
@@ -48,7 +46,10 @@ def __init__(
             resource_name (str): Prometheus API resource name
             client (DynamicClient): Admin client resource
             verify_ssl (bool): Perform SSL verification on query
-            bearer_token (str): Used for query OAuth with API endpoint
+            bearer_token (str, Required): Used for query OAuth with API endpoint, this needs to be created via oc
```

Comment:
`bearer_token` should be set as mandatory in args definition and not as empty string